scripts/src/primazactl/cmd/options/agent.py

```diff
@@ -0,0 +1,73 @@
+import os
+from primazactl.cmd.create.namespace.constants import APPLICATION
+from primazactl.primazaworker.workernamespace import WorkerNamespace
+from primazactl.utils import logger
+from .tenant import Tenant
+from .defaults import defaults
+
+
+class Agent(object):
+
+    type = None
+    cluster_environment = None
+    tenant: Tenant = None
+    name: str = None
+    manifest: str = None
+    version: str = None
+    agent: WorkerNamespace = None
```

Comment:
would it be more appropriate to name this field `worker_namespace`?

```suggestion
    worker_namespace: WorkerNamespace = None
```

Reply:
I think agent works better, the WorkerNamespace object could have a better name.

Reply:
great point. WDYT about `AgentNamespace`?

Reply:
Sounds good, will get to it.

Reply:
Done